src/db.py

```diff
@@ -0,0 +1,132 @@
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, Float, String, MetaData
+import sqlalchemy as sql
+import logging
+import os
+import pandas as pd
+from scipy.stats import mode
+import argparse
+
+Base = declarative_base()
+
+# set up looging config
+logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+DATABASE_NAME = 'instacart'
+SQLITE_DB_STRING = 'sqlite:///data/{}.db'.format(DATABASE_NAME)
+
+conn_type = 'mysql+pymysql'
+user = os.environ.get('MYSQL_USER')
+password = os.environ.get('MYSQL_PASSWORD')
+host = os.environ.get('MYSQL_HOST')
+port = os.environ.get('MYSQL_PORT')
+RDS_DB_STRING = '{}://{}:{}@{}:{}/{}'\
+                .format(conn_type, user, password, host, port, DATABASE_NAME)
+
+
+class OrderType(Base):
+    """Create a data model for order types derived from cluster centroids."""
+    __tablename__ = 'ordertypes'
+
+    label = Column(Integer, primary_key=True)
+
+    # Described by means
+    reordered = Column(Float, unique=False, nullable=False)
+    organic = Column(Float, unique=False, nullable=False)
+    popular = Column(Float, unique=False, nullable=False)
+    prepared = Column(Float, unique=False, nullable=False)
+    dairy = Column(Float, unique=False, nullable=False)
+    gluten = Column(Float, unique=False, nullable=False)
+    snack = Column(Float, unique=False, nullable=False)
+    meat = Column(Float, unique=False, nullable=False)
+    fish = Column(Float, unique=False, nullable=False)
+    beverage = Column(Float, unique=False, nullable=False)
+    veg = Column(Float, unique=False, nullable=False)
+    order_size = Column(Float, unique=False, nullable=False)
+
+    # Described by modes
```

Comment:
why was modes used for this? is it so that the data type would be integer for sure?

Reply:
An average time of day or day of week for example doesn't really make much sense or add value whereas the **most common** time is more interesting.